docs/aks/onboarding-form.md

```diff
@@ -0,0 +1,45 @@
+# Teacher services infra - Service onboarding form
+
+Note - this is a migration of two existing services. They share a single codebase, are served to the public from
+two different domains, and are currently deployed to (non-AKS) Azure.
+
+- Service name? Access your teaching qualifications / Check a teacher's record (two services, single codebase)
+- Domains ?
+  - access-your-teaching-qualifications.education.gov.uk (already exists)
+  - check-a-teachers-record.education.gov.uk (already exists)
+  - access-your-teaching-qualifications.teacherservices.cloud
+  - check-a-teachers-record.teacherservices.cloud
+  - test._ and preprod._ variants of the above.
+- Service short name? aytqctr
+- Repository? https://github.com/DFE-Digital/access-your-teaching-qualifications/
+- Environments? e.g. develoment, staging, production
+  - test
+  - preprod
+  - production
```

Comment:
@felixclack I've opted not to have a dev environment here as it's always seemed a little redundant. Children's barred list took the same approach when it was set up on AKS.